validator/testdata/amp4ads_feature_tests/amp_story_ad_errors.out

```diff
@@ -40,6 +40,20 @@ amp4ads_feature_tests/amp_story_ad_errors.html:31:4 The mandatory attribute 'con
 >>     ^~~~~~~~~
 amp4ads_feature_tests/amp_story_ad_errors.html:34:4 The attribute 'content' in tag 'meta name=amp-cta-landing-page-type' is set to the invalid value 'GAME'. [DISALLOWED_HTML]
 |
+|      <!-- Should fail due to no `content=`. -->
+|      <meta name="amp4ads-vars-foo">
+>>     ^~~~~~~~~
+amp4ads_feature_tests/amp_story_ad_errors.html:37:4 The mandatory attribute 'content' is missing in tag 'meta name=amp4ads-vars-*'. [DISALLOWED_HTML]
+|
+|      <!-- Should fail due unrecognized `name` -->
+|      <meta name="amp4ads-vars" content="bar">
+>>     ^~~~~~~~~
+amp4ads_feature_tests/amp_story_ad_errors.html:40:4 The attribute 'name' in tag 'meta name= and content=' is set to the invalid value 'amp4ads-vars'. [DISALLOWED_HTML]
```

Comment:
Another thing to consider. Will you ever have a case where you want to have a specific rule for the `content` attribute value?

For example, something like: "If `name=amp4ads-vars-visible`, then `content` must have a value of `true` or `false`.". If you imagine something like that, then you should plan for it now, because after this change, such a rule would be making the validator rules *more* strict, which is generally hard to do.

Reply:
Talked to @lannka offline. We will keep this as a generic API as far as validation is concerned, with the understanding that this pushes the responsibility to the runtime to handle any content related contracts.